spec/services/ingest_iati_activities_spec.rb

```diff
@@ -176,5 +175,19 @@
         expect(transaction_without_description.description).to eql("Unknown description")
       end
     end
+
+    context "when a description has escaped characters and extra whitespace" do
+      it "normalizes the text" do
+        _beis = create(:beis_organisation)
+        uksa = create(:organisation, name: "UKSA", iati_reference: "GB-GOV-EA31")
+        legacy_activities = File.read("#{Rails.root}/spec/fixtures/activities/uksa/with_escaped_characters.xml")
+
+        described_class.new(delivery_partner: uksa, file_io: legacy_activities).call
+
+        activity = Activity.find_by(previous_identifier: "GB-GOV-13-GCRF-UKSA_TZ_UKSA-021")
+
+        expect(activity.description).to eql("Both Ethiopia and Kenya are flood & drought prone with significant mortality & economic losses attributed to these events in each country.")
```

Comment:
I like the fact you're testing for the exact string you hope to see as a result of the private method used. This gives me more confidence than seeing the implementation or method repeated as so:

```
expect(activity.description).to eql(CGI.unescape("Both Ethiopia and Kenya are flood & drought prone with significant mortality &amp; economic losses attributed to these events in each country.").strip)
```